sycl/test-e2e/SpecConstants/2020/kernel-bundle-api.cpp

```diff
@@ -9,8 +9,7 @@
 //
 // RUN: %{build} -o %t.out -fsycl-dead-args-optimization
 // RUN: %{run} %t.out
-// FIXME: ACC devices use emulation path, which is not yet supported
-// UNSUPPORTED: accelerator
+//
```

Comment:
Just a nit: do we want to keep this line?

Reply:
I think it looks more readable this way. 😄